Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -3,156 +3,138 @@ import SnowplowTracker
 import Core
 
 final class Analytics {
+    private static var tracker: TrackerController {
+        Snowplow
+            .createTracker(namespace: "ios_sp",
+                           network: .init(endpoint: Environment.current.snowplow),
+                           configurations: [TrackerConfiguration()
+                                                .appId(Bundle.version)
+                                                .platformContext(true)
+                                                .geoLocationContext(true),
```

Comment:
Have you checked with BI whether we need these `platformContext ` and `geoLocationContext `? Have you also checked the other parameters like `exceptionAutotracking `, `installAutotracking` or `lifecycleAutotracking`? just to name a few. Seems there's lots of new stuff that's automatically tracked 🤔 

I think most are [true](https://snowplow.github.io/snowplow-objc-tracker/interface_s_p_tracker_configuration.html) by default, but actually we should set them to false as we don't need those events and can save the bandwidth, don't we?

Reply:
@ecotopian No I didn't check with BI. My scope here is to update to the latest SDK and leave it in a state where it can work as before.

But the points you make are worth exploring and maybe we should set a meeting with BI to see if they have more insights on it, maybe is also worth it to create a separate ticket for that.

Reply:
Yes, good idea! I'll approach them to find out more about the changes.